Neo4j.Driver/Neo4j.Driver/Internal/Result/ResultReaderBuilder.cs

```diff
@@ -85,7 +82,7 @@ public IStatementResultReader PreBuild()
             return _records.Count > 0 ? _records.Dequeue() : null;
         }
 
-        private void SetReceiveOneFunc(Func<Task> receiveOneAction)
+        internal void SetReceiveOneAction(Func<Task> receiveOneAction)
```

Comment:
About naming: 
It is actually a function, so suggesting to name this to `SetReceiveOneFunc`